examples/step-27/step-27.cc

```diff
@@ -612,141 +685,63 @@ namespace Step27
 
   // @sect4{LaplaceProblem::estimate_smoothness}
 
+  // As described in the introduction, we will need to take the maximum
+  // absolute value of fourier coefficients which correspond to $k$-vector
+  // $|{\bf k}|= const$. To filter the coefficients Table we
+  // will use the FESeries::process_coefficients() which requires a predicate
+  // to be specified. The predicate should operate on TableIndices and return
+  // a pair of <code>bool</code> and <code>unsigned int</code>. The latter
+  // is the value of the map from TableIndicies to unsigned int.  It is
+  // be used to define subsets coefficients from which we search for the one
```

Comment:
remove 'be', say 'subsets of coefficients' rather than 'subsets coefficients' (?)